generators/server/templates/src/main/resources/config/liquibase/master.xml.ejs

```diff
@@ -20,13 +20,18 @@
 
     <property name="floatType" value="float4" dbms="postgresql, h2"/>
     <property name="floatType" value="float" dbms="mysql, oracle, mssql, mariadb"/>
-    <%_ if (prodDatabaseType === 'postgresql' || devDatabaseType === 'postgresql') { _%>
+    <%_ if (prodDatabaseType === 'postgresql') { _%>
     <property name="clobType" value="longvarchar" dbms="h2"/>
     <%_ } else { _%>
     <property name="clobType" value="clob" dbms="h2"/>
     <%_ } _%>
-    
     <property name="clobType" value="clob" dbms="mysql, oracle, mssql, mariadb, postgresql"/>
+    <property name="uuidType" value="uuid" dbms="h2"/>
+    <%_ if (prodDatabaseType === 'mysql' || prodDatabaseType === 'mariadb') { _%>
+    <property name="uuidType" value="binary(16)" dbms="<%= prodDatabaseType %>"/>
```

Comment:
`binary(16)` is the most efficient way to store UUID in MySQL, and also what Hibernate does by default.
Unfortunately this is not how Liquibase handle it by default: https://liquibase.jira.com/browse/CORE-3081